app-guides/newbie.md

```diff
@@ -0,0 +1,44 @@
+---
+description: A guide for new crypto users
+---
+
+# 👋 First steps in crypto
+
+## 🔐 Wallet & seed phrase
+
+Before anything, navigating the crypto world happens through wallets that are used to store, receive, and send tokens. Whether you want to buy NFTs or earn yield on your capital, you will need a wallet. 
+
+Usually, a wallet is represented by an application that lets users access and spend their funds through their secret seed phrase.
+
+A seed phrase is a random set of words that are used to uniquely create and identify an owner of tokens on the blockchain. Anyone with access to a seed phrase can spend the associated funds. Remember: always keep your seed phrase private!
+
+A wallet can be used to generate addresses to receive funds, and send transactions on the blockchain. 
+
+If you want to learn more about seed phrase or wallets, you can dive into [Metamask user guides](https://metamask.zendesk.com/hc/en-us/articles/4404722782107-User-Guide-Secret-Recovery-Phrase-password-and-private-keys). 
+
+Now that we know what a wallet is, let's create a wallet :point_down:
+
+
+## 🦊 Creating a wallet with Metamask
+
+Metamask is a wallet that works as a browser extension. You can follow [these steps](https://metamask.zendesk.com/hc/en-us/articles/360015489531-Getting-started-with-MetaMask) to install it. 
+
+When it's done, Metamask will prompt you to "Import a wallet" or "Create a wallet". Click on **Create a wallet** and follow the steps. Don't forget to keep the secret recovery phrase (seed phrase) very safe: ideally not on your phone or computer. 
+
+
+
+## Next steps
+
+### Buying ETH to pay for transaction fees
+
+Now that you have a wallet and your seed phrase is safe, you will need ether (ETH) to pay for transaction fees on the blockchain (called gas). You can read more about that [here](https://metamask.zendesk.com/hc/en-us/articles/360015489611-Learn-the-basics-of-blockchains-and-Ethereum-miners-and-validators-gas-cryptocurrencies-and-NFTs-block-explorer-networks-etc-). 
+
+You can buy ETH directly from Metamask by following [this](https://metamask.zendesk.com/hc/en-us/articles/360058239311-Directly-buying-tokens-with-on-ramps-in-MetaMask
+) guide. You will need at least 0.1 ETH to pay for a few transactions. 
+
+If you have an account at a centralized exchange, you can also buy ETH from there and transfer it to one of your Metamask account. 
+
+### Follow our Angle guides
+
+Once you have ETH in your wallet, you can follow our Angle guides to see how to use the protocol. 
```

Comment:
Let's add a link to a guide. 

Or now that you're setup, you may want to start having fun and getting yield on Euros. Take a look for example at this guide we have made to get agEUR